extension/package.json

```diff
@@ -738,8 +738,8 @@
     "lint": "npm run lint:prettier && npm run lint:eslint",
     "lint:prettier": "prettier --check \"**/*.{ts,js,json,svg,md,yml}\"",
     "lint:eslint": "eslint . --ext .js,.ts",
-    "lint:fix": "npm run lint:fix:prettier && npm run lint:eslint -- --fix",
```

Comment:
`--` is end of command line flag. Please make sure it still works in bash after this change.

Reply:
Just try again, and found we should keep this `--`